src/server/snapshot.h

```diff
@@ -95,8 +95,8 @@ class SliceSnapshot {
   void SerializeEntry(DbIndex db_index, const PrimeKey& pk, const PrimeValue& pv,
                       std::optional<uint64_t> expire, RdbSerializer* serializer);
 
-  // Push StringFile buffer to channel.
-  void PushFileToChannel(DbIndex db_index, io::StringFile* sfile);
+  // Push byte slice to channel.
+  void PushBytesToChannel(DbIndex db_index, io::Bytes bytes);
```

Comment:
My idea:

If I'll be implementing an async writer with the IoBuf swapping technique, then I can use it for single shard snapshots as well instead of a channel with small portions

Reply:
Why is it important? Take into account that iobuf is not optimized for being a queue. It may realloc heavily for variable size workloads. I think it's too soon to start optimizing the code towards a winning approach.